### ISSUE TITLE
Developpement

### ISSUE THREAD

**Issue by carllaliberte** [metaverse-eth/contract#2]
Hi, 
      I'm in contact with Coinbase devs. What could they do to help your project?

**Comment by ant3ffg** [comment]
Better question would be how could they further develop the project ? Currently there is a small community, the name does help however how do we build this further ?

**Comment by carllaliberte** [comment]
That's what i'm trying to figure too

**Comment by ant3ffg** [comment]
Shoot me and email and let's talk further about what you can do anthony@3feetfromgoldllc.com

**Comment by carllaliberte** [comment]
***@***.***é depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  12 h 06  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Shoot me and email and let's talk further about what you can do ***@***.***

**Comment by carllaliberte** [comment]
Can I give you email to a Coinbase personn who knows some Coinbase devs. He is interess in this projectEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  12 h 06  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Shoot me and email and let's talk further about what you can do ***@***.***

**Comment by ant3ffg** [comment]
Why don't you give them my email and have them reach out so we can discuss further.

**Comment by carllaliberte** [comment]
I invest some money in this project and I'm not going to sell. My ERC adress: 0xc29f2789b7abaa03d15bdb4fb24f62578df40ebaThe Coinbase person I've talk to you is thinking about buying around 400 000$ $Meta Tokens. I beleive this should help to kickstart the project.Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  12 h 06  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Shoot me and email and let's talk further about what you can do ***@***.***

**Comment by carllaliberte** [comment]
The Coinbase person would like to know if you are a dev?Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  12 h 06  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Shoot me and email and let's talk further about what you can do ***@***.***

**Comment by carllaliberte** [comment]
I'm in communication with him with What's app. When I will receive his business email address, I will send it to you with pleasureEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  12 h 40  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Why don't you give them my email and have them reach out so we can discuss further.

**Comment by ant3ffg** [comment]
Ok wonderful! Thank you very much.

**Comment by carllaliberte** [comment]
Take a look at Web3inu.org Same contract adress but in Bsc : 0xd0c4bc1b89bbd105eecb7eba3f13e7648c0de38fWhite papers seems to have been made by the same graphic designerBridge with Etherium is foreseen in phase 3I'm confusedEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  14 h 10  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Ok wonderful! Thank you very much.

**Comment by ant3ffg** [comment]
I'll be honest I have no idea where they got the contract from or why. Let's see what your dev friend can do.

**Comment by carllaliberte** [comment]
Yes, he is making research. I will tell youEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  16 h 05  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
I'll be honest I have no idea where they got the contract from or why. Let's see what your dev friend can do.

**Comment by ant3ffg** [comment]
Ok good stuff.

**Comment by carllaliberte** [comment]
Look, I created somme names on Twitter for devsEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  18 h 26  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Ok good stuff.

**Comment by carllaliberte** [comment]
Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  18 h 26  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Ok good stuff.

**Comment by carllaliberte** [comment]
For your information, I boutht around 2.4% of the coins.. it's not a lot, but i'm Holding Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  18 h 26  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Ok good stuff.

**Comment by carllaliberte** [comment]
Maybe you should by some and promot it like meEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  18 h 26  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Ok good stuff.

**Comment by carllaliberte** [comment]
This is the What's app Coinbase personne I've talked to you .. if you want to reach him too :+1 (256) 542-1545I dont't have his name, but mine is CarlEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-10  18 h 26  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Ok good stuff.

**Comment by ant3ffg** [comment]
Carl would it be possible to put us all in a WhatsApp Chat and you make the introduction ? +1-856-630-8797

**Comment by carllaliberte** [comment]
Ok, I will try Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : ant3ffg ***@***.***> Date : 2022-01-11  07 h 44  (GMT-05:00) À : metaverse-eth/contract ***@***.***> Cc : Carl Laliberté ***@***.***>, State change ***@***.***> Objet : Re: [metaverse-eth/contract] Developpememt (Issue #2) 
Carl would it be possible to put us all in a WhatsApp Chat and you make the introduction ? +1-856-630-8797
